widgetssdk/src/main/java/com/glia/widgets/chat/model/history/OperatorChatItem.java

```diff
@@ -2,14 +2,14 @@
 
 import java.util.Objects;
 
-public abstract class OperatorChatItem extends ChatItem {
+public abstract class OperatorChatItem extends ParticipantMessageChatItem {
```

Comment:
This solution is good. However, I have two concerns here. Let me describe them and there is a chance that we could mitigate them.
1. `ParticipantMessageChatItem` name itself doesn't correspond to the underlying idea - "object with message id". Seems like the word "participant"  has nothing to do with message identifier. Maybe we can find a better name like `ChatItemWithId` or `IdentifiedChatItem`, `LinkedChatItem`, `ChatItemWithPayload`, etc.
2. What do you think regarding adding message id to all types of chat items directly (without introducing a new superclass)? What benefit can this idea bring - at first, some ChattItems already contain message id and secondly avoiding new nesting level without a strong may reduce complexity.

But I am not insisting on these suggestions. Just think if they make sense from your point of view.

Reply:
1. Went with LinkedChatItem
2. Because not all times of ChatItems need the ID. For example OperatorStatusItem or the MediaUpgradeStartedTimerItem. And ChatItem id is not the same as the message ID that comes from the Glia Core SDK. ChatItem ID is an internal implementation for Widget SDK and is sometimes linked to Glia Core message ID and sometimes not. Ideally, it should be converted to 2 separate fields but my PR is already pretty big as is